perfmetrics/scripts/ml_tests/pytorch/run_model.sh

```diff
@@ -42,12 +42,12 @@ logging:
     compress: true
 metadata-cache:
   ttl-secs: 1728000
+  stat-cache-max-size-mb: 403
```

Comment:
Assuming 320 bytes is upper bound of one entry size in the cache.

Reply:
yes. Do you want me add a comment or something ?